src/components/Reminder/index.tsx

```diff
@@ -42,29 +52,44 @@ const Notification = styled(Box, {
     background-color: ${
       bordered
         ? theme.colors.transparent
-        : theme.colors[variants[variant].background]
+        : theme.colors[variants[variant].background as Color]
     };
     border: 1px solid ${
       bordered ? theme.colors.gray300 : theme.colors.transparent
     };
     transition: all .3s ease-in-out;
 
     &:hover {
-      box-shadow: 0 3px 6px ${theme.colors[variants[variant].background]};
-      border: 1px solid ${theme.colors[variants[variant].main]};
+      box-shadow: 0 3px 6px ${
+        theme.colors[variants[variant].background as Color]
+      };
+      border: 1px solid ${theme.colors[variants[variant].main as Color]};
     }
   `}
 
   & strong {
     ${({ variant, theme }) => `
-      color: ${theme.colors[variants[variant].main]};
+      color: ${theme.colors[variants[variant].main as Color]};
     `}
   }
 `
 
-const Reminder = ({ text, variant, bordered, to, ...props }) => (
+type ReminderProps = {
+  bordered?: boolean
+  text: string
+  to?: string
+  variant?: Variants
+}
+
+const Reminder: FunctionComponent<ReminderProps> = ({
+  text,
+  variant = 'info',
+  bordered = false,
+  to,
+  ...props
+}) => (
   <Notification
-    as={to ? UniversalLink : 'a'}
+    as={to ? (UniversalLink as React.ElementType) : 'a'}
     type={to ? null : 'button'}
     icon="east"
     style={{}}
```

Comment:
wut ? 😄

Reply:
Otherwise it complains that universalLink isnt a react.elementtype :/

Reply:
I mean the `style={{}}` ^^